hs_file_types/models/timeseries.py

```diff
@@ -449,6 +450,12 @@ def __hash__(self):
             graph.remove((result_node, HSTERMS.timeSeriesResultUUID, series_id))
             graph.add((result_node, HSTERMS.timeSeriesResultUUID, Literal([str(series_id)])))
 
+        # abstract is all by itself on this model, won't get picked up by ingestion automatically
+        description_node = graph.value(subject=subject, predicate=DC.description, default=None)
+        if description_node:
+            self.abstract = graph.value(subject=description_node, predicate=DCTERMS.abstract, default="").toPython()
+        self.save()
```

Comment:
Should ```self.save()``` go within the if condition block since it is only needed when self.abstract is set?

Reply:
It won't have a functional difference, but it's probably best to do so.